### ISSUE TITLE
Typehinting enclosures detects returntype wrongly

### ISSUE THREAD

**Issue by raalderink** [djoos/Symfony-coding-standard#134]
The sniff Symfony.Functions.ReturnType.Invalid detects enclosures wrongly.

I have a function like this:
```php
function test(array &$a): void
{
    foreach ($a as &$b) {
        usort(
            $b,
            function ($c, $d): int {
                return (int)$c['sort'] <=> (int)$d['sort'];
            }
        );
    }
}
```

The function's returntype is void, but the closure's returntype is int. However, the sniff fails on the closure's return line, with the following message: "Use return null; when a function explicitlyreturns null values and use return; when the function returns void values"

**Comment by tifabien** [comment]
I have the same issue.
You can temporarly add `/* @codingStandardsIgnoreLine */` just above your `return (int)$c['sort'] <=> (int)$d['sort'];`

**Comment by djoos** [comment]
Closing this issue - fixed in #141

**Comment by tifabien** [comment]
Great news! Thanks a lot!
